### PR TITLE
fix(condo): Added missing csv-parse package

### PR DESCRIPTION
**Pull request by dkoviazin**

### CODE REVIEW

File: apps/condo/package.json [hunk 1]
```diff
@@ -91,6 +91,7 @@
     "body-parser": "^1.19.1",
     "carbone": "3.2.3",
     "classnames": "^2.2.6",
+    "csv-parse": "^5.5.6",
```
Comment (sitozzz) [inline apps/condo/package.json:94]: It is not a version, that used inside another apps!

**Review by sitozzz — CHANGES_REQUESTED** [review]
I think we should use same version across all of the apps if it possible. If it is not - please comment why we should not do that

File: apps/condo/package.json (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/package.json]: Fixed

**Review by sitozzz — APPROVED** [review]
Now looks good

**Pull request merged**